vsts_ci/azure-pipelines-e2e.yml

```diff
@@ -13,37 +13,21 @@ stages:
 - stage: E2ETest
   displayName: E2E Tests
   jobs:
-  - job: SetupScript
-    steps:
-    - script: |
-        sudo apt-get update
-        sudo apt-get install jq
-      displayName: 'Install jq'
-    - task: AzureCLI@2
-      inputs:
-        azureSubscription: 'Connection to Pipeline resources for IoT Edge Config'
-        scriptType: 'bash'
-        scriptLocation: 'inlineScript'
-        inlineScript: |
-          az upgrade
-          az extension add --name azure-iot
-          az --version
-          az account set -s $(AzureSubscriptionId)
-      displayName: 'Set Azure resources'
-
   - job: LinuxE2ETests

```

Comment:
>job [](http://example.com/codeflow?start=4&length=3)

I think we should keep one job running on vmImage: Ubuntu 18.04 simply for the benefit of not having to run the cleanup step.